ffs/scheduler/scheduler.go

```diff
@@ -593,6 +651,41 @@ func (s *Scheduler) executeColdStorage(ctx context.Context, curr ffs.CidInfo, cf
 	}}, allErrors, nil
 }
 
+func (s *Scheduler) waitForDeals(ctx context.Context, c cid.Cid, startedProposals []cid.Cid) ([]ffs.FilStorage, []ffs.DealError) {
```

Comment:
This is a helper func in the scheduler, if some logic wants to wait for multiple deals... considering now that the API for _ColdStorage_ is `WaitForDeal`.